### DISCUSSION TITLE
Info su allarme Home Assistant

### DISCUSSION THREAD

**Discussion by Maxfox72** [indomus/forum#2017] [HUB: Home Assistant]
Ciao, 
Ho provato ad utilizzare il pannello allarmi già integrato in HA ma mi dice che non trova entità nonostante abbia sensori di movimento e rilevatori di apertura che funzionano egregiamente.
Dove sbaglio ?

**Comment by KC-inDomus** [comment]
> Ciao, Ho provato ad utilizzare il pannello allarmi già integrato in HA ma mi dice che non trova entità nonostante abbia sensori di movimento e rilevatori di apertura che funzionano egregiamente. Dove sbaglio ?

quale pannello allarmi?

**Reply by Maxfox72** [reply]
Ciao 
L'unico che mi viene dato come scelta quando voglio aggiungere una scheda.

**Reply by KC-inDomus** [reply]
se non esiste l'entità alarm_control_panel, come pensi di poterlo mai visualizzare e utilizzare su una scheda?
https://indomus.it/focus/panoramica-i-sistemi-dallarme-la-domotica-e-home-assistant/

**Reply by Maxfox72** [reply]
Ciao,
Ok grazie
